[2025-06-18 02:25:39,366] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table 2021-12-24T00:00:00+00:00 [queued]>
[2025-06-18 02:25:39,412] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table 2021-12-24T00:00:00+00:00 [queued]>
[2025-06-18 02:25:39,413] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 02:25:39,413] {taskinstance.py:1043} INFO - Starting attempt 4 of 6
[2025-06-18 02:25:39,413] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 02:25:39,473] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2021-12-24T00:00:00+00:00
[2025-06-18 02:25:39,491] {standard_task_runner.py:52} INFO - Started process 564 to run task
[2025-06-18 02:25:39,504] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'create_postgres_table', '2021-12-24T00:00:00+00:00', '--job-id', '8323', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/connect_postgres.py', '--cfg-path', '/tmp/tmpv4r32e35', '--error-file', '/tmp/tmpe50thxo3']
[2025-06-18 02:25:39,510] {standard_task_runner.py:77} INFO - Job 8323: Subtask create_postgres_table
[2025-06-18 02:25:39,655] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table 2021-12-24T00:00:00+00:00 [running]> on host d5e6667b8b51
[2025-06-18 02:25:39,785] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-24T00:00:00+00:00
[2025-06-18 02:25:39,786] {postgres.py:69} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2025-06-18 02:25:39,810] {base.py:74} INFO - Using connection to: id: postgres_localhost. Host: postgres, Port: 5432, Schema: test, Login: airflow, Password: XXXXXXXX, extra: None
[2025-06-18 02:25:39,823] {dbapi.py:180} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2025-06-18 02:25:39,831] {dbapi.py:186} INFO - Rows affected: -1
[2025-06-18 02:25:39,837] {postgres.py:73} INFO - NOTICE:  relation "dag_runs" already exists, skipping

[2025-06-18 02:25:39,874] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=create_postgres_table, execution_date=20211224T000000, start_date=20250618T022539, end_date=20250618T022539
[2025-06-18 02:25:39,940] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 02:25:39,959] {local_task_job.py:146} INFO - Task exited with return code 0
